### DISCUSSION TITLE
Whatsapp Flows

### DISCUSSION THREAD

**Discussion by leoparente** [david-lev/pywa#23] [Ideas]
Is there any roadmap to support Whatsapp Flows?

Flows is a new feature introduced last month: https://developers.facebook.com/docs/whatsapp/flows

It seems not trivial to implement it, because it has its own endpoinds like template. However, I think supporting its callbacks as it is a new `interactive object `would be a great start.

Offtopic: I'm just learning about the Cloud API and I find this a great project already.

**Comment by david-lev** [comment]
Hi @leoparente
I saw the feature and it did look interesting, but relatively complicated to implement.
I will try in the next few days to prepare a roadmap for implementing the feature.

In the meantime, API requests can be sent to the required endpoints via the `.api.send_raw_request(...)` method of the WhatsApp client.

Incoming requests can also be handled using the same FastAPI or Flask app provided to pywa.

**Comment by david-lev** [comment]
See #26

**Reply by leoparente** [reply]
Nice work! I will test it

**Reply by david-lev** [reply]
Thanks!
I would love to receive feedback
